### PR TITLE
Spelling and formatting fixes for the Review Committee pattern 

### PR DESCRIPTION
**Pull request by spier**
Copy Patlet of pattern to the overview page, which also removes the non-standard spelling of InnerSource. Also some reformatting of pattern itself.

In detail:

- move Alias block to the end
- Adapted Status to Structured, according to the [new maturity levels](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/contributor-handbook.md)

### CODE REVIEW

**Comment by spier** [comment]
@gruetter would be awesome if you could review these changes.

**Review by lenucksi — APPROVED** [review]
This looks good to me and does not change any semantics. Waiting for @gruetter to click the big green merge button.

**Pull request merged**
